server/src/test/java/io/crate/integrationtests/UserDefinedFunctionsIntegrationTest.java

```diff
@@ -344,4 +349,20 @@ public void test_pg_function_is_visible_when_oid_is_retrieved_from_column() thro
         assertThat(response.rows()[0][0], is(true));
         execute("drop table oid_test");
     }
+
+    @Test
+    public void test_udf_used_inside_generated_column_definition_cannot_be_dropped() {
```

Comment:
I've added unit tests by the latest fixup commit, shall we still keep this for the  full-stack test (testing that the validation code is triggered)?

Reply:
Yep would keep one integration test 👍